Marlin/language_en.h

```diff
@@ -261,6 +261,24 @@
 #ifndef MSG_ESTEPS
 #define MSG_ESTEPS                          "Esteps/mm"
 #endif
+#ifndef MSG_X1OFFSET
+#define MSG_X1OFFSET                         "T1X Offset"
```

Comment:
> On May 28, 2015, at 8:23 PM, Scott Lahteine <notifications@github.com> wrote:
> That's an interesting point around the Marlin G-code, speaking of standards and practices. T is correct for the tool number. However, there's some question whether that's cool with G-code standards.
> 
Which G-code standards? RS274?

3.7.3 Select Tool — T

To select a tool, program T..., where the T number is the carousel slot for the tool. The tool is not changed until an M6 is programmed (see Section 3.6.3).

3.6.3 Tool Change — M6

To change a tool in the spindle from the tool currently in the spindle to the tool most recently selected (using a T word — see Section 3.7.3), program M6.

> Specifically, as I understand it, the G-code guidelines we're following reserve G, M, and T for the main codes, N for line numbers, F for feed-rate, X, Y, and Z for the three primary axes, etc.
> 
> That said, these "standards" are pretty loosely applied in Marlin. The T code is used by M200 for the extruder number. M204 and M205 use it for "travel moves" rate. M206 uses it for "theta." And a few other codes, such as M221, use it for the extruder number.
> 
> But M301 and M303 use E for the extruder number. G29 uses E as a flag for "enhanced G29."
> 
> So, for consistency, T seems best for the extruder number right now.
>